lib/executors/Simulator.ts

```diff
@@ -302,4 +312,19 @@ export class Simulator {
       handleError(error);
     }
   }
+
+  /**
+   * Makes the dashboard page for a simulation that was created with `save` or `save_if_fails`
```

Comment:
*Makes the dashboard page for a simulation publicly viewable*

Is a good enough comment, since we might change the exact name of the props used for this case.